go/flagutil/flagutil.go

```diff
@@ -124,3 +125,58 @@ func (value StringMapValue) String() string {
 	sort.Strings(parts)
 	return strings.Join(parts, ",")
 }
+
+// DualCaseStringListVar cretes a flag which supports both dashes and underscores
+func DualCaseStringListVar(p *[]string, name string, value []string, usage string) {
```

Comment:
Sorry to be nitpicky, but `DualCase` to me means upper/lower case. Here we are talking about two different formats. I suggest renaming these functions to `DualFormat...`

Reply:
Renaming. I used the term "Case" because it's common to refer to this as `snake_case` and `kebab-case`.